server/backend/api/app/constants.py

```diff
@@ -87,3 +89,15 @@ class EmailSendingStatus(str, Enum):
 ERROR_CODE_MISSING_KEY_ATTRIBUTE = "missing_key_attribute"
 ERROR_CODE_INVALID_REQUEST_PARAMETER = "invalid_request_parameter"
 ERROR_CODE_TERMS_CONDITIONS_REQUIRED = "terms_condition_required"
+
+
+# --------------------------------- Schema --------------------------------- #
+USER_NAME_MAX_LEN = 20
+FIRST_NAME_MAX_LEN = 50
+LAST_NAME_MAX_LEN = 50
+EMAIL_MAX_LEN = 250
+CLIENT_NUMBER_MAX_LEN = 8
+CLIENT_NAME_MAX_LEN = 60
+DISTRICT_ID_MAX_LEN = 20  # Assumption
```

Comment:
We need to confirm and not have assumptions. But I'm confused why are we referencing district at all?

Reply:
removed district related stuff https://github.com/bcgov/nr-forests-access-management/pull/1587/commits/91166704341d868ecde8639337f807f003f70b43